biorxiv/polka_subset_experiment/03_rerun_analyses_with_added_subset.py

```diff
@@ -0,0 +1,602 @@
+    .reset_index()
+    .sort_values("count", ascending=False)
+)
+
+preprint_count_df.head()
+# -
+
+# ## PMCOA Token Counts
+
+polka_published = list(Path("output/pmcoa_word_counts").rglob("*tsv"))
+
+# +
+published_count = aggregate_word_counts(polka_published)
+
+published_count_df = (
+    pd.DataFrame.from_records(
+        [
+            {
+                "token": token[0],
+                "pos_tag": token[1],
+                "dep_tag": token[2],
+                "count": published_count[token],
+            }
+            for token in published_count
+        ]
+    )
+    .query(f"token not in {stop_word_list}")
+    .groupby("token")
+    .agg({"count": "sum"})
+    .reset_index()
+    .sort_values("count", ascending=False)
+)
+
+published_count_df.head()
+# -
+
+# ## Get Token Stats
+
+preprint_vs_published = get_term_statistics(preprint_count_df, published_count_df, 100)
+preprint_vs_published.to_csv(
+    "output/updated_preprint_to_published_comparison.tsv", sep="\t", index=False
+)
+preprint_vs_published
+
+full_plot_df = calculate_confidence_intervals(preprint_vs_published)
+full_plot_df.head()
+
+plot_df = (
+    full_plot_df.sort_values("odds_ratio", ascending=True)
+    .iloc[3:]
+    .head(20)
+    .append(full_plot_df.sort_values("odds_ratio", ascending=False).head(20))
+    .assign(
+        odds_ratio=lambda x: x.odds_ratio.apply(lambda x: np.log2(x)),
+        lower_odds=lambda x: x.lower_odds.apply(lambda x: np.log2(x)),
+        upper_odds=lambda x: x.upper_odds.apply(lambda x: np.log2(x)),
+    )
+)
+plot_df.head()
+
+g = (
+    p9.ggplot(
+        plot_df, p9.aes(y="token", x="lower_odds", xend="upper_odds", yend="lemma")
+    )
+    + p9.geom_segment(color="#253494", size=3.5, alpha=0.7)
+    + p9.scale_y_discrete(
+        limits=(plot_df.sort_values("odds_ratio", ascending=True).lemma.tolist())
+    )
+    + p9.scale_x_continuous(limits=(-11, 11))
+    + p9.geom_vline(p9.aes(xintercept=0), linetype="--", color="grey")
+    + p9.annotate(
+        "segment",
+        x=2,
+        xend=8,
+        y=1.5,
+        yend=1.5,
+        colour="black",
+        size=0.5,
+        alpha=1,
+        arrow=p9.arrow(length=0.1),
+    )
+    + p9.annotate("text", label="Published Enriched", x=5, y=2.5, size=12, alpha=0.7)
+    + p9.annotate(
+        "segment",
+        x=-2,
+        xend=-8,
+        y=39.5,
+        yend=39.5,
+        colour="black",
+        size=0.5,
+        alpha=1,
+        arrow=p9.arrow(length=0.1),
+    )
+    + p9.annotate("text", label="Preprint Enriched", x=-5, y=38.5, size=12, alpha=0.7)
+    + p9.theme_seaborn(context="paper", style="ticks", font_scale=1.1, font="Arial")
+    + p9.theme(
+        figure_size=(10, 6),
+        panel_grid_minor=p9.element_blank(),
+    )
+    + p9.labs(y=None, x="Preprint vs Published log2(Odds Ratio)")
+)
+g.save("output/figures/preprint_published_frequency_odds.svg")
+g.save("output/figures/preprint_published_frequency_odds.png", dpi=250)
```

Comment:
Are the arrows in this figure backwards? Looking at the previous table `s.` is found 25 times in `corpus_one_a` and 145 times in `corpus_two_b`. If corpus one is preprints and corpus two is published documents, then `s.` would be enriched in publications, not preprints

Reply:
Great Catch!